rbac/common/manager/rbac_manager.py

```diff
@@ -13,11 +13,90 @@
 # limitations under the License.
 # -----------------------------------------------------------------------------
 
+import logging
+import inspect
+
+from sawtooth_sdk.processor.exceptions import InvalidTransaction
+from rbac.common.protobuf.rbac_payload_pb2 import RBACPayload
 from rbac.common.user.user_manager import UserManager
 from rbac.common.role.role_manager import RoleManager
+from rbac.common.task.task_manager import TaskManager
+
+LOGGER = logging.getLogger(__name__)
+
+
+def walk_filter(value):
+    return str(type(value)).startswith("<class 'rbac.common.")
+
+
+def hasgetattr(value, attrib):
```

Comment:
Since the name is has[...], I would assume by looking at this function that it would return a boolean (similar to the way hasattr does). However, it returns either the result of getattr(), or None. 

Please name it as such. Maybe something like 'get_attr_or_none(value, attr)' 
or attr() to align with message_type_name() below

Reply:
changed to ```getattr_or_none```